R/check.R

```diff
@@ -136,6 +133,28 @@ check <- function(pkg = ".",
   )
 }
 
+can_document <- function(pkg) {
+  required <- pkg$roxygennote
+  if (is.null(required)) {
+    # Doesn't use roxygen2 at all
+    return(FALSE)
+  }
+
+  installed <- packageVersion("roxygen2")
+  if (required != installed) {
```

Comment:
Don't know if this would be better, but maybe something like.

```r
same_version <- required == installed`

if(!same_version) {
 # show message
}

same_version
```

Reply:
I just tried it and I don't think it's an improvement